test/game_teams_manager_test.rb

```diff
@@ -125,4 +125,40 @@ def test_rival
 
     assert_instance_of String, game_teams_manager.rival("6")
   end
+
+  def test_sum_values
+    path = "./data/games.csv"
+    game_manager = GamesManager.new(path)
+
+    key_value_arr = [[1, 1], [2, 2]]
+    expected = {1 => 1, 2 => 2}
```

Comment:
This isn't accurately testing what this method does. We need multiple pairs with the same index[0] so that the method can sum index[1] for each pair that shares index[0].
key_value_arr = [[1, 1], [1, 3], [1, 5], [2, 2], [2, 1]]
    expected = {1 => 9, 2 => 3}